cmd/fleetctl/query.go

```diff
@@ -174,10 +174,9 @@ func queryCommand() *cli.Command {
 						if online > 0 {
 							percentOnline = 100 * float64(responded) / float64(online)
 						}
-					}
-
-					if responded >= online && flExit {
-						return nil
+						if responded >= online && flExit {
+							return nil
+						}
```

Comment:
Given that the `--exit` flag is documented as exiting when 100% of the online hosts have returned results, I think it'd be nicer if the stats had a chance to get printed before returning? I.e. in your example in the description of the PR, it currently outputs:

```
0% responded (0% online) | 0/1 targeted hosts (0/1 online)
```

Reply:
Great idea!